<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>545</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Настройка БУРР-30</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>520</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>3000</width>
     <height>1500</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QTableWidget" name="vmu_param_table">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <column>
         <property name="text">
          <string>Параметр</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Описание</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Адрес</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Значение</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Размерность</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QPushButton" name="connect_vmu_btn">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Подключиться</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="constantly_req_vmu_params">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Постоянно</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Время опроса, мс</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="response_time_edit">
        <property name="inputMask">
         <string extracomment="от 10 до 60 000"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start_record">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Запись</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="select_file_vmu_params">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhMultiLine</set>
        </property>
        <property name="text">
         <string>Выбрать файл  
 с параметрами 
 для записи</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
